app/views/shared/analytics/_non_returning_hypertensive_patients_panel.html.erb

```diff
@@ -12,6 +12,7 @@
              options: { id: 'non-returning-hypertensive-patients-graph' },
              column_class: 'featured-graph-column-yellow',
              graph_data: non_returning_hypertensive_patients_per_month
+                 .select { |month, _| month >= number_of_months.months.ago.at_beginning_of_month }
```

Comment:
What do you think of moving the logic to a view helper or better a `Presenter` object?

Reply:
Agreed. Moved this logic to a helper. 

We don't currently use Presenter objects anywhere yet, and this seems small enough to move to a helper.